plugins/googlecloudplatform/bq-schema/v1.1.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1.6
+FROM --platform=$BUILDPLATFORM golang:1.22.1-bookworm AS build
```

Comment:
```suggestion
FROM --platform=$BUILDPLATFORM golang:1.22.2-bookworm AS build
```

Reply:
Updated in https://github.com/bufbuild/plugins/pull/1150/commits/e73e9c56052bf48907ff5e35afe551340ca38017